Dockerfile

```diff
@@ -70,6 +72,9 @@ RUN \
 	DAAPD_RELEASE=$(curl -sX GET "https://api.github.com/repos/ejurgensen/forked-daapd/releases/latest" \
 	| awk '/tag_name/{print $4;exit}' FS='[""]'); \
  fi && \
+ curl -L https://github.com/mopidy/libspotify-archive/blob/master/libspotify-${LIBSPOTIFY_VERSION}-Linux-${ARCH}-release.tar.gz?raw=true | tar -xzf- -C /tmp/source/ && mv /tmp/source/libspotify* /tmp/source/libspotify && sed -i 's/ldconfig//' /tmp/source/libspotify/Makefile && make -C /tmp/source/libspotify prefix=/tmp/libspotify-build install && rm -rf /tmp/source/libspotify && \
```

Comment:
Can you break this into multiple lines so it's easier to read?
Thanks

Reply:
perhaps it was in testing before, but seems to have moved to community: https://pkgs.alpinelinux.org/packages?name=mxml-dev&branch=edge

Reply:
In the build with community, the repo had a connection error (temporary): https://ci.linuxserver.io/blue/organizations/jenkins/Docker-Pipeline-Builders%2Fdocker-daapd/detail/PR-37/2/pipeline#step-147-log-215

Reply:
It indeed works with community! The CI passes now, thank you for looking into it!